beets/library.py

```diff
@@ -61,9 +60,9 @@ def __init__(self, field, pattern, fast=True, case_sensitive=None):
         """
         super(PathQuery, self).__init__(field, pattern, fast)
 
-        # By default, the case sensitivity depends on the platform.
```

Comment:
This comment is probably still relevant (although perhaps "platform" -> "filesystem"?).